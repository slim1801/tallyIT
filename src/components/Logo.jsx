import React from "react";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faCar } from "@fortawesome/free-solid-svg-icons";

export default function() {
  return (
    <div style={{ color: "white", height: 200, textAlign: "center" }}>
      <FontAwesomeIcon style={{ fontSize: 100 }} icon={faCar} />
      <div style={{ fontSize: 42, marginLeft: 25 }}>Parks and Calc</div>
    </div>
  );
}
