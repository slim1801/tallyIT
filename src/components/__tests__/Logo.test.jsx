import React from "react";
import Logo from "../Logo";

describe("Logo", () => {
  test("smoke test", () => {
    const LogoWrapper = shallow(<Logo />);
    expect(LogoWrapper).toMatchSnapshot();
  });
});
