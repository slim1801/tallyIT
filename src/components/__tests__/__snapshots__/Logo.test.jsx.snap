// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo smoke test 1`] = `ShallowWrapper {}`;
